test/providers/memory.spec.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-unused-expressions */
+const { expect } = require('chai');
+const Memory = require('../../lib/providers/memory');
+
+describe('Memory provider', () => {
+  describe('create method', () => {
+    it('should have a create property', () => {
+      expect(Memory).to.have.a.property('create');
+    });
+
+    it('create should be a function', () => {
+      expect(Memory.create).to.be.a('function');
+    });
+
+    it('create should return the class instance', () => {
+      const memory = Memory.create();
+      expect(memory).to.be.an.instanceOf(Memory);
+    });
+  });
+
+  describe('set, get, has, getKeys, delete methods', () => {
+    const memory = new Memory();
+    const methods = ['set', 'get', 'has', 'getKeys', 'delete'];
+    methods.forEach(method => {
+      it(`should have ${method} method`, () => {
+        expect(Memory.prototype).to.have.a.property(method);
```

Comment:
Consider testing the methods with `respondTo` instead of `property`, to make a test more specific.